kernel/src/sched.rs

```diff
@@ -52,7 +52,9 @@ pub unsafe fn do_process<P: Platform, C: Chip>(
                             process.push_function_call(ccb);
                         }
                         Task::IPC((otherapp, ipc_type)) => {
-                            ipc.schedule_callback(appid, otherapp, ipc_type);
+                            ipc.map(|ipc| {
+                                ipc.schedule_callback(appid, otherapp, ipc_type);
+                            });
```

Comment:
I'm a little wary of silently dropping things, maybe a `map_or` with a `debug_assert!(false, "IPC task on board with no IPC")` or something.

Reply:
But wouldn't that be a way for an app to crash the kernel? Why would we want that?

Reply:
But if all of the IPC system calls just immediately return `ENODEVICE`, then there should never be an IPC task, correct? Then I would be ok with the assert, but it should indicate this is a bug with the kernel and not the app.